examples/outputoptions/typemappings/cfg.yaml

```diff
@@ -0,0 +1,13 @@
+# yaml-language-server: $schema=../../../configuration-schema.json
+package: typemappings
+output: gen.go
+generate:
+  models: true
+output-options:
+  # to make sure that all types are generated, even if they're unreferenced
+  skip-prune: true
+  type-mappings:
+    string-email:
+      define-via-alias: true
```

Comment:
@stevenh got a good example of what this would look like when generated? I don't see any difference right now :thinking: